Platforms/QemuSbsaPkg/PlatformBuild.py

```diff
@@ -295,6 +295,8 @@ def PlatformPostBuild(self):
         args += " ARCH=" + self.env.GetValue("TARGET_ARCH").lower()
         args += " DEBUG=" + str(1 if self.env.GetValue("TARGET").lower() == 'debug' else 0)
         args += " SPM_MM=1 EL3_EXCEPTION_HANDLING=1"
+        args += " ENABLE_FEAT_HCX=1" # Features used by hypervisor

```

Comment:
I think a bit more context about what you're doing here would be helpful to add to the PR description.

Or just a link to something like the description here https://trustedfirmware-a.readthedocs.io/en/latest/getting_started/build-options.html.

Reply:
Also, you might want to elaborate on "Windows fixes" (like a specific build number) in case someone is having issues and wants to repro successfully.

Reply:
Will improve the description. Unfortunately, the Windows build number for the fix hasn't been determined yet as it to the mainline or flighted builds.